tests/subresource_api_test.go

```diff
@@ -237,6 +241,164 @@ var _ = Describe("[sig-compute]Subresource Api", func() {
 				}, 90*time.Second, 1*time.Second).Should(Equal(v1.Running))
 			})
 		})
+
+		Context("ExpandInstancetype endpoint", func() {
+			Context("instancetype", func() {
+				var (
+					instancetype *instancetypev1alpha1.VirtualMachineInstancetype
+					expectedCpu  *v1.CPU
+					vmi          *v1.VirtualMachineInstance
+				)
+
+				BeforeEach(func() {
+					instancetype = newVirtualMachineInstancetype(nil)
+					instancetype.Spec.CPU.Guest = 2
+
+					expectedCpu = &v1.CPU{
+						Sockets: 2,
+						Cores:   1,
+						Threads: 1,
+					}
+
+					var err error
+					instancetype, err = virtCli.VirtualMachineInstancetype(util.NamespaceTestDefault).
+						Create(context.Background(), instancetype, metav1.CreateOptions{})
+					Expect(err).ToNot(HaveOccurred())
+
+					vmi = libvmi.NewCirros()
```

Comment:
There's no reason to specifically use `NewCirros()` right? I think the approach for such tests is just to use `New()` now as per https://github.com/kubevirt/kubevirt/pull/8117

Reply:
Done